source/retryable-reads/tests/aggregate.yml

```diff
@@ -0,0 +1,96 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+    
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "Aggregate succeeds on first attempt"
+        clientOptions:
+            retryReads: true
+        operation: &retryable_operation
+            name: aggregate
+            object: collection
+            arguments:
+                pipeline:
+                    - $match: {_id: {$gt: 1}}                  
+                    - $sort: {x: 1}                        
+                batchSize: 2
```

Comment:
Setting a batchSize here causes an extraneous getMore command so I think we should remove the `batchSize` for these tests. *Note* the same problem exists in the *find* tests. 

We should also have a test that an error on a getMore is not retried for both aggregate and find.
```suggestion
```